modules/olm-updating-index-image.adoc

```diff
@@ -35,20 +35,43 @@ endif::[]
 ----
 $ opm index add \
     --bundles <registry>/<namespace>/<new_bundle_image>@sha256:<digest> \//<1>
-    --from-index <registry>/<namespace>/<existing_index_image>:<tag> \//<2>
-    --tag <registry>/<namespace>/<existing_index_image>:<tag> \//<3>
-    --pull-tool podman <4>
+    --from-index <registry>/<namespace>/<existing_index_image>:<existing_tag> \//<2>
+    --tag <registry>/<namespace>/<existing_index_image>:<updated_tag> \//<3>
+    --pull-tool podman //<4>
+----
+<1> The `--bundles` flag specifies a comma-separated list of additional bundle images to add to the index.
+<2> The `--from-index` flag specifies the existing index that was previously pushed.
+<3> The `--tag` flag specifies the image tag that you want the updated index image to have.
```

Comment:
I don't know if "apply" is the right way to put this, but it seems it's less about wanting a tag and more about the tag you want being implemented?
```suggestion
<3> The `--tag` flag specifies the image tag to apply to the updated index image.
```